adamsjr8576/palette-picker-fe
# Delete branch

### What does this PR do?
- Add new actions: `deletePaletteFromStore` | `deleteProjectFromStore`  | `deletePaletteByProjectId`
- These use already existing reduces, just adds new cases
- Add a new icon for deleting a project 
- Add delete requests in `apiCalls`
- Moves the `GET` request in projectCard to `apiCalls`
- Removes a project and palette from the db and the store

### How to test?
- Pull down
- Run `npm start`
- Remove a project by clicking the trash can
- Check the redux dev tools to see it removed 
- Check the console for a message that it is been removed from the db

### Issues:
- This PR closes no issues
- I opened one issue, #1 
- I have updated the board 

### Notes:
- Here is the new format:
![Screen Shot 2020-02-11 at 1 57 32 PM](https://user-images.githubusercontent.com/48968224/74278436-75208800-4cd6-11ea-9077-99ff03013eaa.png)

Thanks!
@adamsjr8576
